File: app/src/main/java/com/permutassep/ui/FragmentLoginSignUp.java
package com.permutassep.ui;

import android.app.ActionBar;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.lalongooo.permutassep.databinding.CaFragmentLoginSignupBinding;

public class FragmentLoginSignUp extends BaseFragment {

    private CaFragmentLoginSignupBinding binding;

    public static FragmentLoginSignUp newInstance() {
        return new FragmentLoginSignUp();
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        binding = CaFragmentLoginSignupBinding.inflate(inflater, container, false);

        ActionBar actionBar = getActivity().getActionBar();
        if (actionBar != null) {
            actionBar.hide();
        }
        setupOnClickListeners();
        return binding.getRoot();
    }

    private void setupOnClickListeners() {
        binding.btnRegister.setOnClickListener(v -> navigationListener.onNextFragment(FragmentSignUp.class));
        binding.btnLogin.setOnClickListener(v -> navigationListener.onNextFragment(FragmentLogin.class));
        binding.tvContinue.setOnClickListener(v -> navigationListener.onNextFragment(FragmentPagedNewsFeed.class));
    }
}

File: app/src/main/java/com/permutassep/ui/FragmentLoginSignUp.kt
package com.permutassep.ui

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.compose.ui.platform.ComposeView
import androidx.compose.ui.platform.ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed
import com.permutassep.ui.compose.WelcomeScreen

class FragmentLoginSignUp : BaseFragment() {

    override fun onCreateView(
            inflater: LayoutInflater,
            container: ViewGroup?,
            savedInstanceState: Bundle?
    ): View? {
        return ComposeView(requireContext()).apply {
            setViewCompositionStrategy(DisposeOnViewTreeLifecycleDestroyed)
            setContent {
                WelcomeScreen()
            }
        }
    }

//    @OnClick(R.id.btnRegister)
//    fun onBtnRegisterClick() {
//        navigationListener.onNextFragment(FragmentSignUp::class.java)
//    }
//
//    @OnClick(R.id.btnLogin)
//    fun onBtnLoginClick() {
//        navigationListener.onNextFragment(FragmentLogin::class.java)
//    }
//
//    @OnClick(R.id.tvContinue)
//    fun onTvContinueClick() {
//        navigationListener.onNextFragment(FragmentPagedNewsFeed::class.java)
//    }

    companion object {
        @JvmStatic
        fun newInstance(): FragmentLoginSignUp {
            return FragmentLoginSignUp()
        }
    }
}